cmd/root.go

```diff
@@ -70,11 +78,33 @@ var viewCmd = &cobra.Command{
 			}
 			var ss hivev1.SelectorSyncSet
 			ss = pkg.CreateSelectorSyncSet(args[0], selector, resources, patches)
-			j, err := json.MarshalIndent(&ss, "", "    ")
-			if err != nil {
-				log.Fatalf("error: %v", err)
-			}
-			fmt.Printf("%s\n\n", string(j))
+			fmt.Printf("%s\n\n", GetOutputStr(ss))
 		}
 	},
 }
+
+// GetOutputStr returns the output as json or yaml string
+func GetOutputStr(ss hivev1.SelectorSyncSet) string {
+	var b []byte
+	var err error
+	switch o := output; o {
+	case "yaml":
+		b, err = yaml.Marshal(&ss)
+	default:
+		b, err = json.MarshalIndent(&ss, "", "    ")
+	}
+	if err != nil {
+		log.Fatalf("error: %v", err)
+	}
+	return string(b)
+}
+
+// Contains tells whether array a contains member x.
+func Contains(a []string, x string) bool {
```

Comment:
Contains -> contains

Reply:
it should probably be in some public helper, but will change for now.